cloud/new_ec2.go

```diff
@@ -338,6 +376,11 @@ func (m *ec2Manager) TerminateInstance(h *host.Host) error {
 	if m.spot(h) {
 		canTerminate, err := m.cancelSpotRequest(h)
 		if err != nil {
+			grip.Error(message.WrapError(err, message.Fields{
+				"msg":     "error terminating spot instance",
+				"host_id": h.Id,
+				"distro":  h.Distro.Id,
+			}))
 			return errors.Wrap(err, "error canceling spot request")
```

Comment:
should we wrap the error before logging?

Reply:
I'm not sure what you mean. Don't use `message.WrapError`?